android/src/main/java/io/invertase/firebase/messaging/RNFirebaseMessaging.java

```diff
@@ -64,6 +64,11 @@ public void requestPermission(Promise promise) {
     promise.resolve(null);
   }
 
+  @ReactMethod
+  public void registerForRemoteNotifications(Promise promise) {
```

Comment:
We can remove this and add an if (isAndroid) return null; line in the JS method, native code changes not required then 👍